02_pipelinerun/01_build_deploy_api_pipelinerun.yaml

```diff
@@ -7,7 +7,7 @@ spec:
     name: build-and-deploy
   params:
   - name: deployment-name
-    value: vote-api
+    value: pipelines-vote-api
   - name: git-url
     value: https://github.com/openshift/pipelines-vote-api.git
   - name: IMAGE
```

Comment:
We should probably rename this, too. In triggers scenario, we set it to `image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/$(tt.params.git-repo-name)` so if the user follows the docs end-to-end, they will end up with 4 image streams